audit/test/Integration.t.sol

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+import {Poke} from "src/Poke.sol";
+import {RegistryReceiverV0} from "src/RegistryReceiverV0.sol";
+import {EthPlaysV0} from "src/EthPlaysV0.sol";
+
+contract IntegrationTest is Test {
+    // Test storage
+    Poke poke;
+    RegistryReceiverV0 registryReceiver;
+    EthPlaysV0 ethPlays;
+
+    address deployer;
+    address constant alice = address(1);
+    address constant bob = address(2);
+    address constant charlie = address(3);
+
+    // EthPlays parameters
+    uint256 alignmentVoteCooldown;
+    uint256 chaosVoteReward;
+    uint256 orderDuration;
+    uint256 chaosInputRewardCooldown;
+    uint256 orderInputReward;
+    uint256 chaosInputReward;
+    uint256 controlAuctionDuration;
+    uint256 controlDuration;
```

Comment:
IIRC some linter advises specifying the visibility (public, private, internal). I can never remember what the default is.